packages/components/src/components/modals/Modal/Modal.tsx

```diff
@@ -231,8 +233,9 @@ const Modal = ({
     const areStepsShown =
         totalProgressBarSteps !== undefined && currentProgressBarStep !== undefined;
 
+    // Model has intentionally always Elevation = 1 (it resets the elevation)
     return (
-        <>
+        <ElevationContext baseElevation={1}>
```

Comment:
just quick note - probably not related to this PR: 

background + shadow on darkmode is weird and hardly visible

<img width="612" alt="image" src="https://github.com/trezor/trezor-suite/assets/858321/3330a3c2-28cc-41a6-8d04-0698045b8b6b">


lightmode is fine:
<img width="770" alt="image" src="https://github.com/trezor/trezor-suite/assets/858321/dfbb13a3-d63a-4a21-b1ac-4eada3ed3b35">

Reply:
leo will talk about this with the designer